React/Base/RCTUIKit.h

```diff
@@ -482,6 +482,30 @@ NS_INLINE CGRect CGRectValue(NSValue *value)
 @end
 #endif // ]TODO(macOS GH#774)
 
+// RCTUIActivityIndicatorView
+
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
+#define RCTUIActivityIndicatorView UIActivityIndicatorView
+#else
+@interface RCTUIActivityIndicatorView : UIActivityIndicatorView
+
+@property (nonatomic, assign) UIActivityIndicatorViewStyle activityIndicatorViewStyle;
+@property (nonatomic, assign) BOOL hidesWhenStopped;
+@property (nullable, readwrite, nonatomic, strong) RCTUIColor *color;
+@property (nonatomic, readonly, getter=isAnimating) BOOL animating;
+
+- (void)startAnimating;
+- (void)stopAnimating;
+
+// RCTUISwitch
+
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
+#define RCTUISwitch UISwitch
+#else
+@interface RCTUISwitch : NSSwitch
+@end
+#endif // ]TODO(macOS GH#774)
+
```

Comment:
I think I messed this up when attempting to resolve a diff conflict. Could you update the branch and resolve again? May help with CI failing.

Reply:
fixed in 91d707454721ba4055606864561324022940959c